cmd/dep/init.go

```diff
@@ -209,6 +227,45 @@ func hasImportPathPrefix(s, prefix string) bool {
 	return strings.HasPrefix(s, prefix+"/")
 }
 
+// getVersionConstituents extracts version constituents
+func getVersionConstituents(v gps.Version) (version, revision string) {
```

Comment:
This is actually really close to the `VersionComponentStrings()` func in gps. If we were to go this route, I think I'd rather see that function reused. But I think another route might be better, as I'll describe below...